src/app/template-library/template-library-home/template-library-home.component.ts

```diff
@@ -35,7 +36,7 @@ export class TemplateLibraryHomeComponent implements OnInit {
         for (const category of this.categories) {
           this.categorisedTemplates.push({
             category: category,
-            templates: this.templates.filter(template => template.type === category.name)
+            templates: this.templates.filter(template => UtilsService.removeAllSpecialCharactersAndToLower(template.type) === UtilsService.removeAllSpecialCharactersAndToLower(category.name))
```

Comment:
This is not the correct way we use util service, please check other components as an example

Reply:
@shawnm0705 as in instantiate the class and call it from the instance rather than expose a static method?